community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -283,13 +283,16 @@ public Relationship getRelationshipById( long id )
         }
```

Comment:
These ID bounds checks should be removed and left to the `StorageEngine`, so we can remove the `MAX_RELATIONSHIP_ID` and `MAX_NODE_ID` constants. The constants in `IdType` that feed our `IdGenerators` should also be removed, and we should source that data from the `RecordFormat` somehow.